src/views/tickets/TicketDetail/TicketDetail.vue

```diff
@@ -93,13 +92,20 @@ export default {
         {
           key: this.$t('tickets.status'),
           value: this.object.status,
-          callback: function(row, data) {
-            const open = vm.$t('common.Open')
-            const close = vm.$t('common.Close')
-            if (data === 'open') {
-              return <el-button type='primary' size='mini'>{open}</el-button>
+          formatter: (item, val) => {
+            if (this.object.status === 'open') {
```

Comment:
const status = {
  approve: {type: primary, title: this.$t('tickets.Approved')},
  reject: ...
  
}